### PR TITLE
DrawNodeV2 0.95.1

### PR DESCRIPTION
**Pull request by aismann**
## Describe your changes

Short info about this PR:
Its the next draft 4 preview test

- All methods should work correct now (e.g. Thickness)
- Memory is more optimized 
- some new tests on cpp test (also few eye catcher)

### For each PR
- [ ] Add Copyright if it missed:   
      - `"Copyright (c) 2019-present Axmol Engine contributors (see AUTHORS.md)."`
- [x] I have performed a self-review of my code.
       
   Optional:
   - [ ] I have checked readme and add important infos to this PR.
   - [ ] I have added/adapted some tests too.
          
### For core/new feature PR
- [ ] I have checked readme and add important infos to this PR.
- [ ] I have added thorough tests.

### CODE REVIEW

**Comment by aismann** [comment]
@halx99 
Can be merged.
It will be great if it part of the Release 2.1.5.

**Comment by aismann** [comment]
Ready for merge now.

File: extensions/README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
 - Version: 4.5
 - License: MIT
 
-## DrawNodeEx
+## DrawNodeEx => DrawNodeV2
 - Upstream: https://github.com/axmolengine/axmol
-- Version: 0.9
+- Version: still in progress...
```
Comment (halx99) [inline extensions/README.md]: please just display the actual version, thanks

Comment (aismann) [reply extensions/README.md]: Ok...numbers only

**Pull request merged**